### ISSUE TITLE
We need a data repository

### ISSUE THREAD

**Issue by nspilman** [junior-dev-struggle-bus/juniordevstrugglebus#30]
Ideally a hosted service to store all of our resources, organization details, etc. Our website(s) can then make calls to the service and get the data as needed from anywhere on the web.

**Comment by AvianAnalyst** [comment]
Were you wanting SQL or NoSQL? 

If SQL do you think it will stay small enough SqLite is good enough? Or do you want to go bigger with something like PostgreSQL? 

For hosting do you have any ideas/plans on what service to use for that?

**Comment by nspilman** [comment]
I'm thinking microservices. The specifics are still not entirely clear, but I think they'll flesh themselves out as we build.

**Comment by Xoadra** [comment]
Looks like @nspilman and I have a solution we're working on implementing for this issue.

Our strategy is to add a Netlify function connected to the site that we'll be able to make calls to for adding resources.  Configuring the database for taking in resource links is what @nspilman is working on using the FaunaDB Netlify addon.  The database will then be hooked into the function I'm creating for writing submissions to the resource database.

We'll also need to build a form into the resources page to submit resources, replacing the off-site redirect to a google form that's currently storing submitted resource links to @nspilman's google sheet.

In short, our strategy is as follows:

- [ ] Configure the FaunaDB database via its Netlify addon: @nspilman 
- [ ] Set up the Netlify function to be callable from the website: @Xoadra 
- [ ] Integrate database access in the Netlify function: @nspilman @Xoadra 

This can be expanded upon in the future to support other data storage needs, but this will get us started and help resolve one core issue with the website at present.

**Comment by Xoadra** [comment]
I've gotten the Netlify function to return some fetched data on the website.  It's now ready for a PR once @nspilman sets up the branch for our joint operation.  🙂 

While working on this though, I encountered a few caveats.  The short of it is we will have to reconsider how we enable people to contribute to the site without potentially compromising the site itself.  I'll explore this scenario in detail when I make my PR, so consider this a heads up for what's to come.

**Comment by nspilman** [comment]
I need to get FaunaDb authing correctly.
